webroot/src/components/Users/UserInvite/UserInvite.vue

```diff
@@ -0,0 +1,66 @@
+<!--
+    UserInvite.vue
+    CompactConnect
+
+    Created by InspiringApps on 11/11/2024.
+-->
+
+<template>
+    <Card class="invite-user-container">
+        <div class="invite-user-title">{{ $t('account.inviteNewUser') }}</div>
+        <MockPopulate :isEnabled="isMockPopulateEnabled" @selected="mockPopulate" />
+        <form @submit.prevent="handleSubmit">
+            <div class="invite-user-form-row">
+                <InputText :formInput="formData.email" class="invite-user-meta invite-email" />
+            </div>
+            <div class="invite-user-form-row">
+                <InputText :formInput="formData.firstName" class="invite-user-meta invite-first-name" />
+            </div>
+            <div class="invite-user-form-row">
+                <InputText :formInput="formData.lastName" class="invite-user-meta invite-last-name" />
+            </div>
+            <div class="invite-user-form-row">
+                <InputSelect :formInput="formData.compact" class="permission-type-select" />
+                <div class="input-connector"></div>
+                <InputSelect :formInput="formData.compactPermission" class="permission-select" />
+            </div>
+            <TransitionGroup name="fade">
+                <div
+                    v-for="(formInput, index) in permissionStateInputs"
+                    :key="formInput.id"
+                    class="invite-user-form-row"
+                >
+                    <div v-if="index === 0" class="row-separator"></div>
+                    <InputSelect :formInput="formData[`state-option-${index}`]" class="permission-type-select" />
```

Comment:
Iirc we didn't want to use back ticks in our templating layer

Reply:
I guess theres not a much better way to do this here though

Reply:
Yeah, this one needs the `index` itself from the `v-for` inside the template. If it needed something that wasn't computed in the template itself, I would 100% agree. I'm open to alternatives; my reasoning here was it seemed a straightforward way to handle an indeterminate number of form inputs that were user-controlled.